### DISCUSSION TITLE
Yann Serra Tutorial page - disk format table does not fit and scroll bar is at bottom

### DISCUSSION THREAD

**Discussion by zorkiii** [keirf/greaseweazle#456] [General]
Any chance the table on the page linked here can be modified to fit without a scroll bar? https://github.com/keirf/greaseweazle/wiki/Yann-Serra-Tutorial

Viewing in Chrome and Edge the end extensions are cut off, then when scrolling to the right, the description of the line is then hidden which is a bit difficult.

![image](https://github.com/keirf/greaseweazle/assets/53704364/260c8f9a-a490-4c98-99e0-97d69f18fa20)

**Comment by keirf** [comment]
I'm not really sure how useful the table is. Perhaps we could at least remove a column or two?
